snapshot/lib/snapshot/options.rb

```diff
@@ -82,6 +82,11 @@ def self.available_options
                                      description: "Enabling this option will automatically erase the simulator before running the application",
                                      default_value: false,
                                      is_string: false),
+        FastlaneCore::ConfigItem.new(key: :disable_spellcheck_on_simulator,
```

Comment:
Negative booleans are not ideal, could you switch it around and call it `enable...`